src/components/MaterialUI/Inputs/CheckboxProps.js

```diff
@@ -5,13 +5,17 @@ class CheckboxProps extends ComponentProps {
 		update: "update",
 		value: "value",
 		label: "label",
+		readOnly: "readOnly",
+		checked: "checked",
```

Comment:
Why did you add "checked" as "value" is already there ?

Reply:
Seems that checked and value it's different props. Value is for the value checkbox represents, but checked for Boolean value of that value

Reply:
@Anstsiya, yes, but for the Wrapper, value was already fulfilling that purposes, we don't need that duplication.

Reply:
@joclefrancois  but don't we need to know what value was changed and put it into a handler to have it in edit mode?

Reply:
Furthermore, it is a breaking changes for any application using it already.

Reply:
Removed checked so we don't add possible breaking changes. We should have further discussion on why we need the checked attribute.